Assets/Scripts/Bird.cs

```diff
@@ -114,19 +117,34 @@ private void OnCollisionEnter2D(Collision2D collision)
         if(ready)
         {
             ready = false;
-            StartCoroutine(ResetAfterDelay());
+            StartCoroutine(ResetAfterDelay(15));
         }
         
     }
 
-    private IEnumerator ResetAfterDelay()
+    private bool EntitiesAreMoving()
     {
-        yield return new WaitForSeconds(3);
+        Entity[] allEntities = GameObject.FindObjectsOfType<Entity>();
+        foreach (Entity e in allEntities)
```

Comment:
You could use a Linq statement if you like

Reply:
Done: https://github.com/cjh15hub/terribletweeters/pull/1/commits/bf5906843a21d2e978aa0e82765b2f23f2a3315e#diff-c873074d36a3026960706dbe9c6d43acef6e4f0c053e0bc1962b4d9e1bd7716fR130